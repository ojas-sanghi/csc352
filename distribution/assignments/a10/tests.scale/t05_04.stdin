28289
86481
46853
16162
70457
