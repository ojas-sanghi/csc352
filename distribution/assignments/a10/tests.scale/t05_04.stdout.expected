28289 (audit=1)
scaled by 1: 28289 (audit=1)
scaled by 2: 56578 (audit=1)
scaled by 3: 169734 (audit=1)
scaled by 4: 678936 (audit=1)
scaled by 5: 3394680 (audit=1)
scaled by 6: 20368080 (audit=1)
scaled by 7: 142576560 (audit=1)
scaled by 8: 1140612480 (audit=1)
scaled by 9: 10265512320 (audit=1)
86481 (audit=1)
scaled by 1: 86481 (audit=1)
scaled by 2: 172962 (audit=1)
scaled by 3: 518886 (audit=1)
scaled by 4: 2075544 (audit=1)
scaled by 5: 10377720 (audit=1)
scaled by 6: 62266320 (audit=1)
scaled by 7: 435864240 (audit=1)
scaled by 8: 3486913920 (audit=1)
scaled by 9: 31382225280 (audit=1)
46853 (audit=1)
scaled by 1: 46853 (audit=1)
scaled by 2: 93706 (audit=1)
scaled by 3: 281118 (audit=1)
scaled by 4: 1124472 (audit=1)
scaled by 5: 5622360 (audit=1)
scaled by 6: 33734160 (audit=1)
scaled by 7: 236139120 (audit=1)
scaled by 8: 1889112960 (audit=1)
scaled by 9: 17002016640 (audit=1)
16162 (audit=1)
scaled by 1: 16162 (audit=1)
scaled by 2: 32324 (audit=1)
scaled by 3: 96972 (audit=1)
scaled by 4: 387888 (audit=1)
scaled by 5: 1939440 (audit=1)
scaled by 6: 11636640 (audit=1)
scaled by 7: 81456480 (audit=1)
scaled by 8: 651651840 (audit=1)
scaled by 9: 5864866560 (audit=1)
70457 (audit=1)
scaled by 1: 70457 (audit=1)
scaled by 2: 140914 (audit=1)
scaled by 3: 422742 (audit=1)
scaled by 4: 1690968 (audit=1)
scaled by 5: 8454840 (audit=1)
scaled by 6: 50729040 (audit=1)
scaled by 7: 355103280 (audit=1)
scaled by 8: 2840826240 (audit=1)
scaled by 9: 25567436160 (audit=1)
